HerPortal.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -11,380 +11,383 @@ public static class LocalAuthorityData
 {
     /// If Custodian Codes change (through LAs merging etc.) we need to ensure
     /// consistency with the HUG2 Public Website data, and consider any remapping that may be required.
-    /// <see href="https://github.com/UKGovernmentBEIS/desnz-home-energy-retrofit-beta/blob/develop/HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs"> HUG2 Public Website codebase </see>
+    /// <see
+    ///     href="https://github.com/UKGovernmentBEIS/desnz-home-energy-retrofit-beta/blob/develop/HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs">
+    ///     HUG2 Public Website codebase
+    /// </see>
```

Comment:
Was this autoformatting?

Reply:
Yes. It also put it like this:

HUG2
Public Website codebase 

as well, but after I pulled the Public Website up a line, it would still revert to having the see tag broken up, but would allow HUG2 Public Website codebase all on one line. 

Should I leave it as it was originally, with all of the <see> tag and its contents on one line?

Reply:
Not super bothered. Slight preference for how it was originally, but whatever auto-formatter thinks.